data/interfaces/default/settings.html

```diff
@@ -813,6 +813,16 @@ <h3>Server Monitoring</h3>
                                         <input type="hidden" class="form-control" id="pms_update_distro" name="pms_update_distro">
                                     </div>
                                 </div>
+                                <div class="row">
+                                    <div class="col-md-2">
+                                        <label for="pms_update_notify_interval">Notify Interval</label>
+                                        <input type="text" class="form-control" data-parsley-type="integer" id="pms_update_notify_interval" name="pms_update_notify_interval" value="${config['pms_update_notify_interval']}" size="3" data-parsley-range="[1,999]" data-parsley-trigger="change" data-parsley-errors-container="#pms_update_notify_interval_error" required>
```

Comment:
I don't think there should be an artificial limit of `999` hours, just a minimum of `1` hour.

```
data-parsley-min="1"
```